Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:carbon
```

@Zooloo2014 If possible could you please use alpine:

```suggestion
FROM node:carbon-alpine
```

The base node carbon image is huge :| If alpine doesn't perhaps `slim`

---

This project is using node-gyp; alpine doesn't have all the dependencies

---

Maybe we can install the required modules into the alpine version for this to work witout having to build the bulky carbon image. Maybe something like this. Havent tested this before so if it doesnt work we can leave it as is.
https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#node-gyp-alpine

---

The above comment is no longer applicable as a better approach has been used by instead using the `node-slim` version which includes the `node-gyp` module for building of certain packages